plans.go

```diff
@@ -67,6 +67,18 @@ const (
 	TrafficPlanLimitUnitPB TrafficPlanLimitUnit = "PB"
 )
 
+type UnitPlanLimit struct {
```

Comment:
Why we should name this like `UnitPlanLimit`, maybe it should be `BackupPlanLimit`?

Reply:
I thought we would have another limits with `units` unit.